README.md

```diff
@@ -1,95 +1,78 @@
-Hardware
-========
+# Pixhawk Hardware Designs
```

Comment:
@LorenzMeier @bkueng This is new entry point to the repo, and needs proper review. At high level it takes the time to explain the licencing (from our PX4 docs) and what an open design actually delivers.  It also splits out the different types of hardware here.

NOTE, this isn't all hardware - it is design. It might make sense to move this repo into its own github org "Pixhawk".